2025-07-24 17:27:06,020 - clientes.aura.registro.registro_login - ERROR - Traceback (most recent call last):
  File "C:\Users\PC\PYTHON\AuraAi2\clientes\aura\registro\registro_login.py", line 26, in registrar_blueprints_login
    safe_register_blueprint(
  File "C:\Users\PC\PYTHON\AuraAi2\clientes\aura\__init__.py", line 75, in safe_register_blueprint
    app.register_blueprint(blueprint, name=unique_name, **kwargs)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\scaffold.py", line 50, in wrapper_func
    return f(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\app.py", line 997, in register_blueprint
    blueprint.register(self, options)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 365, in register
    deferred(state)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 431, in <lambda>
    lambda s: s.add_url_rule(
              ^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 110, in add_url_rule
    self.app.add_url_rule(
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\scaffold.py", line 50, in wrapper_func
    return f(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\app.py", line 1059, in add_url_rule
    raise AssertionError(
AssertionError: View function mapping is overwriting an existing endpoint function: login_blueprint.google_login

2025-07-24 17:27:47,947 - clientes.aura.registro.registro_login - ERROR - Traceback (most recent call last):
  File "C:\Users\PC\PYTHON\AuraAi2\clientes\aura\registro\registro_login.py", line 26, in registrar_blueprints_login
    safe_register_blueprint(
  File "C:\Users\PC\PYTHON\AuraAi2\clientes\aura\__init__.py", line 75, in safe_register_blueprint
    app.register_blueprint(blueprint, name=unique_name, **kwargs)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\scaffold.py", line 50, in wrapper_func
    return f(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\app.py", line 997, in register_blueprint
    blueprint.register(self, options)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 365, in register
    deferred(state)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 431, in <lambda>
    lambda s: s.add_url_rule(
              ^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 110, in add_url_rule
    self.app.add_url_rule(
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\scaffold.py", line 50, in wrapper_func
    return f(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\app.py", line 1059, in add_url_rule
    raise AssertionError(
AssertionError: View function mapping is overwriting an existing endpoint function: login_blueprint.google_login

2025-07-24 17:33:00,592 - clientes.aura.routes.admin_modulos.registro_dinamico_frontend.blueprint - ERROR - Error al cargar información del módulo tareas: 'current_user' is undefined
2025-07-24 17:37:08,473 - clientes.aura.registro.registro_login - ERROR - Traceback (most recent call last):
  File "C:\Users\PC\PYTHON\AuraAi2\clientes\aura\registro\registro_login.py", line 26, in registrar_blueprints_login
    safe_register_blueprint(
  File "C:\Users\PC\PYTHON\AuraAi2\clientes\aura\__init__.py", line 75, in safe_register_blueprint
    app.register_blueprint(blueprint, name=unique_name, **kwargs)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\scaffold.py", line 50, in wrapper_func
    return f(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\app.py", line 997, in register_blueprint
    blueprint.register(self, options)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 365, in register
    deferred(state)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 431, in <lambda>
    lambda s: s.add_url_rule(
              ^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 110, in add_url_rule
    self.app.add_url_rule(
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\scaffold.py", line 50, in wrapper_func
    return f(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\app.py", line 1059, in add_url_rule
    raise AssertionError(
AssertionError: View function mapping is overwriting an existing endpoint function: login_blueprint.google_login

2025-07-24 17:40:15,711 - clientes.aura.registro.registro_login - ERROR - Traceback (most recent call last):
  File "C:\Users\PC\PYTHON\AuraAi2\clientes\aura\registro\registro_login.py", line 26, in registrar_blueprints_login
    safe_register_blueprint(
  File "C:\Users\PC\PYTHON\AuraAi2\clientes\aura\__init__.py", line 75, in safe_register_blueprint
    app.register_blueprint(blueprint, name=unique_name, **kwargs)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\scaffold.py", line 50, in wrapper_func
    return f(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\app.py", line 997, in register_blueprint
    blueprint.register(self, options)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 365, in register
    deferred(state)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 431, in <lambda>
    lambda s: s.add_url_rule(
              ^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 110, in add_url_rule
    self.app.add_url_rule(
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\scaffold.py", line 50, in wrapper_func
    return f(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\app.py", line 1059, in add_url_rule
    raise AssertionError(
AssertionError: View function mapping is overwriting an existing endpoint function: login_blueprint.google_login

2025-07-24 17:45:05,410 - clientes.aura.registro.registro_login - ERROR - Traceback (most recent call last):
  File "C:\Users\PC\PYTHON\AuraAi2\clientes\aura\registro\registro_login.py", line 26, in registrar_blueprints_login
    safe_register_blueprint(
  File "C:\Users\PC\PYTHON\AuraAi2\clientes\aura\__init__.py", line 75, in safe_register_blueprint
    app.register_blueprint(blueprint, name=unique_name, **kwargs)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\scaffold.py", line 50, in wrapper_func
    return f(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\app.py", line 997, in register_blueprint
    blueprint.register(self, options)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 365, in register
    deferred(state)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 431, in <lambda>
    lambda s: s.add_url_rule(
              ^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 110, in add_url_rule
    self.app.add_url_rule(
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\scaffold.py", line 50, in wrapper_func
    return f(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\app.py", line 1059, in add_url_rule
    raise AssertionError(
AssertionError: View function mapping is overwriting an existing endpoint function: login_blueprint.google_login

2025-07-24 17:49:01,532 - clientes.aura.registro.registro_login - ERROR - Traceback (most recent call last):
  File "C:\Users\PC\PYTHON\AuraAi2\clientes\aura\registro\registro_login.py", line 26, in registrar_blueprints_login
    safe_register_blueprint(
  File "C:\Users\PC\PYTHON\AuraAi2\clientes\aura\__init__.py", line 75, in safe_register_blueprint
    app.register_blueprint(blueprint, name=unique_name, **kwargs)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\scaffold.py", line 50, in wrapper_func
    return f(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\app.py", line 997, in register_blueprint
    blueprint.register(self, options)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 365, in register
    deferred(state)
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 431, in <lambda>
    lambda s: s.add_url_rule(
              ^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\blueprints.py", line 110, in add_url_rule
    self.app.add_url_rule(
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\scaffold.py", line 50, in wrapper_func
    return f(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PC\PYTHON\AuraAi2\venv311\Lib\site-packages\flask\app.py", line 1059, in add_url_rule
    raise AssertionError(
AssertionError: View function mapping is overwriting an existing endpoint function: login_blueprint.google_login

